src/InfoViz/Native/HistogramSelector/score.js

```diff
@@ -126,7 +126,13 @@ export default function init(inPublicAPI, inModel) {
       if (!scoreData.name) {
         scoreData.name = `${scoreData.selection.partition.variable} (partition)`;
       }
-      model.provider.setAnnotation(scoreData);
+      if (!passive) {
+        model.provider.setAnnotation(scoreData);
+      } else if (model.provider.isA('AnnotationStoreProvider') && model.provider.getStoredAnnotation(scoreData.id)) {
+        model.provider.updateStoredAnnotations({
+          [scoreData.id]: scoreData,
+        });
+      }
```

Comment:
Not passive means this change triggers it becoming the active annotation. 
But a passive update still will update the stored annotation, without it becoming active? If I'm reading it correctly, I think it makes sense, but add a comment to the passive case?